vnext/Shared/Modules/NetworkingModule.cpp

```diff
@@ -261,7 +261,7 @@ void NetworkingModule::NetworkingHelper::SendRequest(
   int64_t requestId = ++s_lastRequestId;
 
   // Enforce supported args
-  assert(responseType == "text" || responseType == "base64");
+  assert(responseType == "text" || responseType == "base64" || responseType == "blob");
```

Comment:
>|| responseType == "blob" [](http://example.com/codeflow?start=60&length=25)

I think you you've added the blob support to HttpModule, not to NetworkingModule ? (I may be wrong)

Reply:
That is correct, but `NetworkingModule` can still be used by setting a specific runtime option.
It won't implement Blob functionality, but this avoids an unnecessary assert.

Context: This feature began development before switching to `HttpModule`.
Also, `NetworkingModule` will be removed soon.